### PR TITLE
[86] Instagram bar not full width + footer links fix + update map url

### PR DESCRIPTION
**Pull request by terrianx**
on the connect page. event page has an ig bar too but its already full width. tested on different screen sizes
Screenshot 2023-12-23 at 2 04 54 PM

ways to connect, updated to point to next steps. although i dont know where the interest form is so its blank for now
Screenshot 2023-12-23 at 2 05 12 PM

url pointed to maps.app which loaded a bit slower than the url in the footer link. changed it to be more consistent
Screenshot 2023-12-23 at 2 06 09 PM

### CODE REVIEW

File: frontend/src/components/instaBar.js
Comment (terrianx) [inline frontend/src/components/instaBar.js:1]: made relaxed since the vertical alignment was off

before
Screenshot 2023-12-23 at 2 10 23 PM

after
Screenshot 2023-12-23 at 2 10 02 PM

File: frontend/src/components/page-connect/map-details.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const MapDetails = () => {
             <SecondaryButtonLink
               hasArrow={true}
               className={"bg-Shades-0"}
-              to="https://maps.app.goo.gl/K7Z126sj78wQjuGY7"
+              to="https://www.google.com/maps/place/Harvest+Mission+Community+Church/@42.2816338,-83.7372209,17z/data=!4m5!3m4!1s0x883cae6a77eef201:0xaf4019d9fc7aec8e!8m2!3d42.2816359!4d-83.7371982?hl=en&shorturl=1"
```
Comment (xumaple) [inline frontend/src/components/page-connect/map-details.js:25]: why did you change this?

**Review by xumaple — COMMENTED** [review]
Everything else looks good. just wondering why we made the link longer - the old one still worked for me

File: frontend/src/components/page-connect/map-details.js (hunk 1 above)
Comment (terrianx) [reply frontend/src/components/page-connect/map-details.js:25]: yea they both work but the shorter one redirects to the longer one and takes slightly longer to load. i figured we might as well just do the longer one since thats the same one as the footer

File: frontend/src/components/footer.js [hunk 2]
```diff
@@ -72,7 +75,6 @@ const Footer = () => {
                   {connectList.map((item, index) => (
                     <a
```
Comment (thedomcab) [inline frontend/src/components/footer.js:76]: can we use gatsby Link for internal links that points to the gatsby site?

Comment (thedomcab) [inline frontend/src/components/footer.js:76]: Can we use Link gatsby component for internal links and just use normal anchor (a) for external links

File: frontend/src/components/instaBar.js [hunk 3]
```diff
@@ -1,13 +1,14 @@
 import * as React from "react";
 import { InstaIcon } from "./svgs";
 
+// could make entire bar clickable to link to ig page?
```
Comment (thedomcab) [inline frontend/src/components/instaBar.js:4]: this is a phil question.  message him in teams about this.

File: frontend/src/components/footer.js (hunk 2 above)
Comment (terrianx) [reply frontend/src/components/footer.js:76]: yes im going to merge this pr first and then in my links refactor pr ill update this

File: frontend/src/components/instaBar.js (hunk 3 above)
Comment (terrianx) [reply frontend/src/components/instaBar.js:4]: ive messaged him, there is a separate issue for this. https://github.com/hmcc-global/hmccaa-web/issues/128

**Review by thedomcab — APPROVED** [review]
